src/js/static.js

```diff
@@ -22,8 +22,8 @@ gpii.express.router["static"].init = function (that) {
     that.staticHandler = express["static"](that.options.content);
 };
 
-gpii.express.router["static"].route = function (that, req, res) {
-    that.staticHandler(req, res);
+gpii.express.router["static"].route = function (that, req, res, next) {
+    that.staticHandler(req, res, next);
```

Comment:
This change looks good, but I'm wondering why this fault wasn't picked up by the existing tests that you have for staticRouter in router-tests.js? Would it be possible to adjust the tests so that they could guard against a regression here?